.github/workflows/main-release.yml

```diff
@@ -0,0 +1,73 @@
+name: Main Release
+on:
+  push:
```

Comment:
If you want to only trigger on changes to a specific path:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore